### DISCUSSION TITLE
Stopping Codespace. Wont load my codespace anymore

### DISCUSSION THREAD

**Discussion by JasonDambro** [community/community#29088] [Codespaces]
This is the message i was getting and no it wont open my codespace?

VSCodeWorkbench mount failed with "Failed to connect to relay url Code: 1006 Reason: "

How do i rectify this issue. Any help would be greatly appreciated.

**Comment by jfullstackdev** [comment]
is this the first time you encountered this ? 

is your computer low on memory ? how about Internet connection ? antivirus or firewall ?

**Reply by CyriacBr** [reply]
I have the exact same issue.  
My codespace suddenly stopped working and now it is showing "Stopping codespace..." whenever I try to open it. And sometimes I get the message "Failed to connect to relay url Code: 1006 Reason: ".

**Comment by JasonDambro** [comment]
Thanks for your time in getting back to me.

It does drop out from time to time, has been doing so for a while.  Although today was an exception, it would load eventually and display VS code however was not functioning properly as it was displaying unusual messages in the terminal window.  I tried on another computer also and it made no difference. The internet connection is stable.

At first when this started happing the error message i got was:

"VSCodeWorkbench mount failed with "Failed to connect to relay url Code: 1006 Reason: "

Could this be a server issue?

Kind regards

Jason


________________________________

**Comment by 0wwafa** [comment]
same problem here. I have a codespace completely stuck. showing Stopping codespace.

**Comment by Ldh2024** [comment]
Same problem here. It loads and immediately tells me : stopping codespace

**Comment by rbkdigital** [comment]
Facing same issue.

**Comment by thomassiard** [comment]
The error code 1006 typically points to a network or relay connection issue. Here are a few things you can try:

Check Your Network Connection: Codespaces relies on a stable internet connection. If you’re on a restrictive network (like at a school or office), it could be blocking the relay URL. Try connecting from a different network or a VPN to see if that helps.

Update VS Code and Extensions: Ensure that both your VS Code application and the GitHub Codespaces extension are updated to the latest versions. Sometimes, compatibility issues can cause connectivity problems.

Clear Cache and Restart: Clearing the cache in VS Code or even restarting your machine can sometimes fix temporary connection issues.

Contact GitHub Support: If the issue persists, it may be on GitHub’s side, especially if Codespaces is experiencing downtime. You can check [GitHub Status](https://www.githubstatus.com/) to see if there are ongoing issues with Codespaces. If not, reaching out to GitHub Support might be the best step to resolve this.

**Comment by valentinaatm** [comment]
same here also, what do we do?????

**Comment by Aditya141208** [comment]
I can't open it as well it says : stopping codespace

**Comment by Aditya141208** [comment]
I don't know why is this happening, I have a cs50 codespace since I am doing cs50x, also the same codespace was working well in the morning when I logged into it to finish my problem set. If anybody has the fix please comment how to do so I will Highly appreciate it.

**Comment by Aditya141208** [comment]
![Screenshot 2024-11-05 173809](https://github.com/user-attachments/assets/f6207004-1880-495d-a442-fd49df9b2b3d)

**Reply by Aditya141208** [reply]
I just tried to open the codespace in the vs code app and it appears to be working properly

**Reply by Follen-cry** [reply]
have you sorted by any means? or it just magically works after a period of time (how long in your case?) , I run into the exact same issue as you, everytime I clicked to open the codespace it goes "setting codespace" directly to "stopping codespace"...

**Comment by Twist3dS0uL** [comment]
This was working fine until a couple of hours ago. I was logged out and upon logging back in via the browser, I received the message, "Stopping codespace..." and when I click 'Open in VS Code Desktop, the progress bar is stuck 20% on "Setting up remote connection: Shutting down codespace... "

I tried restarting the codespace- it didn't work. 
I have deleted all 4 entries in the Codespace user secrets (cs50_lang, cs50_remote_addr, cs50_token, cs50_tz), recreated them and tried logging in via the browser/VS Code Desktop again however the same issue is persisting.

**Comment by liskin** [comment]
I think there are two issues here:

1. The Codespace sometimes takes a long time to stop. My hypothesis is that this depends on how much storage is used — if there's dozens of gigs of data in the Codespace, it can take over 10 minutes to stop the Codespace.
2. The UX is a bit confusing. Everything looks like the Codespace is stopped, but any attempt to start it back again will result in this "Stopping Codespace" screen. The https://github.com/codespaces/ page should make it clear that the Codespace isn't stopped but is in the process of stopping. It could even provide an estimate if my hypothesis is correct—a few mins for every 10 gigs of storage.

**Comment by avijit-shihaan** [comment]
my codespace suddenly stopped and not opening anymore, even I cant export my un committed changes to another branch. Any help?

**Comment by adrian-85** [comment]
Open for two years and no response?  My laptop un-expectdly shut down, and the codespace had trouble re-loading.  I tried rebuilding, fully rebuilding, wouldn't load any files.  So I stopped it, and now I have the same issue everyone else here mentioned.  Seems like it got into a corrupted state, and GitHub does not have a method to recover.  Guessing my stashed changes are gone, fortunately I just comitted yesterday, so not too much lost.

User beware!

**Comment by ghost** [comment]
![image](https://github.com/user-attachments/assets/125af64c-0805-4ba6-93a8-0b85a69ed885)

Hey folks! I know it sounds a bit stupid, but have you all tried simply restarting it? I was facing the same issue, but the above restart button helped me so much! My codespace opened in literal seconds! 
![image](https://github.com/user-attachments/assets/e8c05182-2c95-4652-b79f-af738d0d3416)
 VS code (and coding in general) is quite new to me, and I am working with CS50 here, so it might just be that the restart button is avaibale only for CS50, but hey, if you find one, pls try it!

**Reply by CouchMunchies** [reply]
Tried it- still stuck. :/

**Reply by Jarek191** [reply]
it is still not working

**Comment by Jarek191** [comment]
![image](https://github.com/user-attachments/assets/437d9518-703c-4a71-93b8-cb622aaa0d58)
 what can I do any Hellp ?

**Reply by YvodeRooij** [reply]
Same issue here!

**Reply by aetloh** [reply]
same

**Reply by nidercalole** [reply]
same

**Comment by zederis** [comment]
I have same issue. Though I can create a new codespace... So maybe if I clicked on "Export Changes To a Branch" I would be able to pick up unsaved changes in a new codespace... ideally i would like to use same codespace..

**Reply by nidercalole** [reply]
In my case i hat to recreate any unpushed changes of the broken codespace.
I could not fix this problem and i also could not export my changes because my codespace was stuck in a Stoping state. i tryed tor restart, force stop, etc via github cli.
Nothing worked.

**Reply by inspi-writer001** [reply]
wow, so.. it's still a dead end?
I have the exact same issue since yesterday,  I even tried updating the machine type
it doesn't work

**Reply by nidercalole** [reply]
for me it definitely was a dead end.
U can try to reach Github support to somehow get the changes...

**Reply by theaustinhatfield** [reply]
I found the solution. If you wait long enough you'll eventually get a restart codespace button, press that and it will work.

**Reply by pooja-muruganandham** [reply]
> I found the solution. If you wait long enough you'll eventually get a restart codespace button, press that and it will work.

how long did you wait?

**Comment by matt-mahdieh** [comment]
Any update here? I am facing the same issue!!

**Reply by aetloh** [reply]
Try on different devices and it should work again in like 1-10h
At least that was it for me

**Comment by gajeshladhar** [comment]
<img width="737" height="660" alt="image" src="https://github.com/user-attachments/assets/ec1495dd-0ec8-40cf-86ca-dc9c8ec001fc" />

stuck from last 1hr, any updates anyone, it has high sensitive data need to be recovered?

**Comment by wefinvest** [comment]
i ahve same issue with my code space as well.. anyone found what is the reason?

**Comment by benjaminclark177-lab** [comment]
It sounds like a relay or network connectivity issue, so checking your internet connection, firewall, or recreating the Codespace may help. If you're already troubleshooting similar development workflows, using an [seo audit platform for agencies](https://lemaudit.com/use-cases/agencies) to keep technical projects organized can also be useful.

**Comment by zwozaiden-glitch** [comment]
somehow i make it to open codespace,i was doing fork for pc cloud https://github.com/jephersonRD/PC-Free,then when i setup already my cloud close and my codespace unactive,few minute i try configure out and i check pull request,there codespace request it ,but you need few minute or you wait on codespace for a minute 
<img width="765" height="399" alt="Screenshot 2026-08-29 23 21 49" src="https://github.com/user-attachments/assets/84c1b2a1-5f9b-4640-8510-c1546f8ada84" />
until look like screenshoot then restart it and then done,i dont know if this really work because i am just dumb ah coders
